shanoir-ng-datasets/src/main/java/org/shanoir/ng/studycard/dto/QualityCardResultEntry.java

```diff
@@ -35,6 +35,8 @@ public class QualityCardResultEntry {
 	
 	private QualityTag tagSet;
 
+	private boolean faildValid = false; // if tag VALID was to put but conditions failed
```

Comment:
Hi @julien-louis, very minor typo: "failed"Valid would be great please. Thank you!